docs/getting-started/best-practices/configuring-microsoft-dfs-with-octopus-server.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Configuring Microsoft DFS with Octopus Server
+description: Guidelines and recommendations for configuring Microsoft DFS as the shared file system for Octopus Deploy.
+position: 120
+hideInThisSection: true
+---
+
+Microsoft DFS is a [distributed file system](https://en.wikipedia.org/wiki/Clustered_file_system#Distributed_file_systems), which is important to consider when configuring Octopus Deploy with a DFS file share. Octopus Server makes specific assumptions about the performance and consistency of the file system when accessing log files, performing log retention, storing deployment packages and other deployment artifacts and temporary storage when communicating with Tentacles.
+
+**DFS in the standard configuration (ie, accessed through a DFS Namespace Root) is _not_ suitable for use as a shared file store with Octopus Deploy.**
+
+Operating Octopus Deploy with the non-recommended DFS configuration will likely result in intermittent and potentially significant issues.
```

Comment:
Suggest adding a call-out (warning), as this seems pretty important to highlight
```suggestion
:::warning
**DFS in the standard configuration (i.e., accessed through a DFS Namespace Root) is _not_ suitable for use as a shared file store with Octopus Deploy.**

Operating Octopus Deploy with the non-recommended DFS configuration will likely result in intermittent and potentially significant issues.
:::